docs/diagrams/ModelClassDiagram.puml

```diff
@@ -36,15 +39,16 @@ ModelManager -right-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
 AddressBook *--> "1" UniquePersonList
+UniquePersonList .right-> UniquePersonListDeserializer
```

Comment:
Should UniquePersonListDeserializer be included? It seems like a less-important class that can be omitted for comprehensibility.

Reply:
Removed.